api/src/main/java/org/openmrs/module/eptsreports/reporting/reports/SetupMISAUKeyPopsReport.java

```diff
@@ -0,0 +1,109 @@
+/*
+ * The contents of this file are subject to the OpenMRS Public License
+ * Version 1.0 (the "License"); you may not use this file except in
+ * compliance with the License. You may obtain a copy of the License at
+ * http://license.openmrs.org
+ *
+ * Software distributed under the License is distributed on an "AS IS"
+ * basis, WITHOUT WARRANTY OF ANY KIND, either express or implied. See the
+ * License for the specific language governing rights and limitations
+ * under the License.
+ *
+ * Copyright (C) OpenMRS, LLC.  All Rights Reserved.
+ */
+package org.openmrs.module.eptsreports.reporting.reports;
+
+import static org.openmrs.module.reporting.evaluation.parameter.Mapped.mapStraightThrough;
+
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Properties;
+import org.openmrs.module.eptsreports.reporting.library.cohorts.GenericCohortQueries;
+import org.openmrs.module.eptsreports.reporting.library.datasets.LocationDataSetDefinition;
+import org.openmrs.module.eptsreports.reporting.library.datasets.MISAUKeyPopsDataSetDefinition;
+import org.openmrs.module.eptsreports.reporting.reports.manager.EptsDataExportManager;
+import org.openmrs.module.eptsreports.reporting.utils.EptsReportUtils;
+import org.openmrs.module.reporting.ReportingException;
+import org.openmrs.module.reporting.report.ReportDesign;
+import org.openmrs.module.reporting.report.definition.ReportDefinition;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+@Component
+public class SetupMISAUKeyPopsReport extends EptsDataExportManager {
+
+  private MISAUKeyPopsDataSetDefinition mISAUKeyPopsDataSetDefinition;
+
+  private GenericCohortQueries genericCohortQueries;
+
+  @Autowired
+  public SetupMISAUKeyPopsReport(
```

Comment:
Good stuff, this what am referring to in the other comments, please replicate uniformity

Reply:
Understood.